core/Tests/util-tests.vala

```diff
@@ -314,20 +441,24 @@ void add_daterange_tests () {
 void add_datetime_tests () {
     Test.add_func ("/Utils/DateTime/is_all_day_false", test_is_all_day_false);
     Test.add_func ("/Utils/DateTime/is_all_day_true", test_is_all_day_true);
+    Test.add_func ("/Utils/DateTime/datetimes_to_icaltime", test_datetimes_to_icaltime);
 }
 
 int main (string[] args) {
     print ("\n");
     var original_tz = Environment.get_variable ("TZ");
     Environment.set_variable ("TZ", "America/Chicago", true);
     print ("Setting $TZ environment variable: " + Environment.get_variable ("TZ") + "\n");
+    ICal.Timezone tz = ICal.Timezone.get_builtin_timezone ("America/Chicago");
+    Calendar.TimeManager.setup_test (tz);
+    print ("Setting up TimeManager with system timezone America/Chicago");
     print ("Starting utils tests:\n");
 
     Test.init (ref args);
```

Comment:
mhm, seems like this is similar to the `setUp` from JUnit, right? So there should probably be some way to access test-related variables/settings?

Reply:
By my understanding, `Test.init` is used to initialize a test suite, not a single test. 

There does appear to be some setup/teardown functionality in the `add` function, but it's quite opaque and under-documented. I'll have to see if I can get it to work in Vala. It seems to depend on some weird parts of C.